.gitignore

```diff
@@ -48,6 +48,8 @@ config/database.yml
 .rbenv-gemsets
 .floo
 .flooignore
+Gemfile
```

Comment:
Gemfile and Gemfile.lock should not be ignored. They are used for consistency in gems and gem versions used in the project. Incase you made changes to the gemfile that you do not want to be checked out, when adding files to staging on git do not include the gemfile ( dont know if i am making sense )